samples/batch.js

```diff
@@ -93,11 +91,10 @@ function createQueryPartitions(instanceId, databaseId, identifier, projectId) {
     .catch(err => {
       console.error('ERROR:', err);
     });
-  // [END create_query_partitions]
+// [END spanner_batch_client]
 }
 
 function createReadPartitions(instanceId, databaseId, identifier, projectId) {
-  // [START create_read_partitions]
```

Comment:
If this region tags is being deleted, and therefore the sample not shown in the docs, we should delete the sample